.metadata/idl/state.idl

```diff
@@ -3,6 +3,19 @@ import "oaidl.idl";
 import "propidl.idl";
 // import "wtypes.idl";
 
+typedef struct StateData
```

Comment:
If this struct is removed, and getbool() returns BOOLEAN, bindgen can correctly reference windows crate BOOLEAN type.